exercises/practice/hello-world/.docs/instructions.md

```diff
@@ -0,0 +1,16 @@
+# Instructions
+
+The classical introductory exercise.
+Just say "Hello, World!".
+
+["Hello, World!"][hello-world] is the traditional first program for beginning programming in a new language or environment.
+
+The objectives are simple:
+
+- Modify the provided code so that it produces the string "Hello, World!".
+- Run the test suite and make sure that it succeeds.
```

Comment:
This seems to be optional, if using the online editor and submitting it that way.  Does the distinction matter for those working locally and submitting via the CLI as opposed to those fully online without local development?

Reply:
I don't quite understand what the problem is, could you be a bit more descriptive?

Reply:
Running the test suite may not be required by the programmer that is creating the solution.  This is something that the platform will do for them if they are using the online editor on our platform.

Indeed, if they do not run the tests at all while submitting from the command line, they can do so even if it never ran, but even if it does not succeed.

Reply:
Ah, yes, if you are doing an online tutorial this will happen on its own, I took these articles from hello world for java because I am trying to get used to the exercise format.

and yes, what you say is quite possible.